packages/apollo-cli/src/__mocks__/localfs.ts

```diff
@@ -0,0 +1,12 @@
+import {Volume, createFsFromVolume} from 'memfs';
+import {patchFs} from 'fs-monkey';
+
+export const vol = Volume.fromJSON({});
+export const fs = createFsFromVolume(vol);
+
+export function withGlobalFS<T>(thunk: () => T): T {
```

Comment:
This patch is needed to handle libraries like `glob`, which directly use `fs`. For those situations we wrap the library call in `withGlobalFS` to patch the `fs` library with our in-memory volume for the duration of the call.